src/pages/MyIssueDetails.js

```diff
@@ -84,7 +90,10 @@ export default function MyIssueDetails({
                     aria-labelledby="milestone"
                     type="text"
                     defaultValue={myIssue.milestone}
-                    onChange={e => (myIssue.milestone = e.target.value)}
+                    onChange={e => {
+                      myIssue.milestone = e.target.value;
+                      handleEditIssue(myIssue);
```

Comment:
This is not really new code and I cant be 100% sure, but it feels like `myIssue` is state or a prop and you are mutating it here. It might make sense to call `handleEditIssue` with a copy of `myIssue`:
```suggestion
                      const milestone = e.target.value;
                      handleEditIssue({ ...myIssue, milestone });
```